File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //prompts the user for a number from 1 through 8
    int height = 0;
    
    while (height <= 0 || height >= 9)
    {
        height = get_int("Height: ");
    }
    
    for (int i = 0; i < height; i++)
    {
        // prints spaces before the #
        for (int s = height - i ; s > 1 ; s--)
        {
            printf(" ");
        
        }
        
        // prints the # 
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        
        printf("\n");

    }
}

File: pset6/mario0/mario.py
from cs50 import get_int

height = 0

# prompts the user for a number from 1 through 8
while height <= 0 or height >= 9:
    height = get_int("Height: ")

for i in range(height):
    
    # prints spaces before the #
    for s in range(height - i - 1):
        print(' ', end='')
    
    # prints the '#' 
    for j in range(i + 1):
        print('#', end='')
    
    print()